ci/travis_jobs/docker_setup.sh

```diff
@@ -3,12 +3,13 @@ echo 'doing docker build'
 # Note: adding --build-arg <arg-name> without any value tells docker to
 #  use value from local environment (export DO_GIT_CLONE)
 
-#${TRAVIS_BUILD_DIR}/ci/travis_jobs/get_data_volumes.py
+${TRAVIS_BUILD_DIR}/ci/travis_jobs/get_data_volumes.py
 
 
 echo Timing docker pull...
 start_seconds=$SECONDS
 
+echo "$DOCKER_PASSWORD" | docker login --username "$DOCKER_USERNAME" --password-stdin
```

Comment:
This is also not necessary. The login command right before the push command (on line 33) is sufficient.

Reply:
Done